ui/src/app/dashboard/controllers/DashCtrl.js

```diff
@@ -2,15 +2,14 @@
 
   angular
     .module('app')
-    .controller('DashCtrl', [ '$scope', 'ToastService', 'DashService',
+    .controller('DashCtrl', [ '$scope', 'ToastService', 'DashService', 'UserService',
       DashCtrl
     ]);
 
-  function DashCtrl($scope, ToastService, DashService) {
+  function DashCtrl($scope, ToastService, DashService, UserService) {
     $scope.dlink = {link: ''};
     $scope.downloads = [];
-    $scope.username = localStorage.getItem('Username');
-
+    $scope.username = UserService.getUsername();
 
     var socket = io.connect('http://localhost:5000/progress');
```

Comment:
Have declarations initially, then initialization codes followed by logic. It is a good practice :) And noticed there were few inconsistencies with indentation formatting. Some places you have used 2 spaces and in several places you have used 4/8 spaces.